config/olm-install/kustomization.yaml

```diff
@@ -1,2 +1,27 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
 resources:
 - install-resources.yaml
+
+namespace: mygatekeeper
+
+patches:
+- patch: |-
+    kind: CatalogSource
+    metadata:
+      name: gatekeeper-operator
+    spec:
+      image: quay.io/gatekeeper/gatekeeper-operator-bundle-index:latest
+  target:
+    kind: CatalogSource
+    name: gatekeeper-operator
+- patch: |-
+    kind: Subscription
+    metadata:
+      name: gatekeeper-operator-sub
+    spec:
+      channel: "3.11"
```

Comment:
Do we have `sed` for` channel: "3.11"`

Reply:
Yes--it's in the `deploy-using-olm` target and was updated to change this `kustomization.yaml` file in this PR.

Reply:
Why not set it to `stable` by default?

Reply:
Works for me! I was just taking the current value from the YAML.